lib/common/photoprism_album_manager.dart

```diff
@@ -48,17 +48,17 @@ class PhotoprismAlbumManager {
     print("Creating new album");
     photoprismModel.photoprismLoadingScreen
         .showLoadingScreen("Creating new album..");
-    var status =
+    String status =
         await Api.createAlbum('New album', photoprismModel.photoprismUrl);
 
-    if (status == 0) {
+    if (status == "-1") {
```

Comment:
@spaibs does this change make sense? Or should I check for "0" here

Reply:
It doesn't matter here.